ale_linters/python/pyls.vim

```diff
@@ -0,0 +1,21 @@
+" Author: aurieh <me@aurieh.me>
+" Description: A language server for Python
+
+call ale#Set('python_pyls_executable', 'pyls')
+
+function! ale_linters#python#pyls#GetExecutable(buffer) abort
+    return ale#Var(a:buffer, 'python_pyls_executable')
+endfunction
+
+function! ale_linters#python#pyls#GetLanguage(buffer) abort
+    return 'python'
+endfunction
+
+call ale#linter#Define('python', {
+\   'name': 'pyls',
+\   'lsp': 'stdio',
+\   'executable_callback': 'ale_linters#python#pyls#GetExecutable',
+\   'command_callback': 'ale_linters#python#pyls#GetExecutable',
+\   'language_callback': 'ale_linters#python#pyls#GetLanguage',
```

Comment:
I'll add support at some point for just writing `'language': 'python'`. I can see that coming up quite a bit.

Reply:
Yeah, that would be nice. Is there anything else I should do?

Reply:
Nah, it's fine.